Cargo.toml

```diff
@@ -1,18 +1,20 @@
 [package]
 name = "hbbft"
 version = "0.1.0"
-authors = ["Vladimir Komendantskiy <komendantsky@gmail.com>",
-           "Andreas Fackler <AndreasFackler@gmx.de>",
-           "Peter van Nostrand <jnz@riseup.net>",
-           "Andrew Gross <andogro@gmail.com>",
-           "Nick Sanders <nsan1129@gmail.com>",
-           "Marc Brinkmann <git@marcbrinkmann.de>"]
-description = "Honey Badger Byzantine fault tolerant consensus algorithm"
+authors = [
+    "Vladimir Komendantskiy <komendantsky@gmail.com>",
+    "Andreas Fackler <AndreasFackler@gmx.de>",
+    "Peter van Nostrand <jnz@riseup.net>",
+    "Andrew Gross <andogro@gmail.com>",
+    "Nick Sanders <nsan1129@gmail.com>",
+    "Marc Brinkmann <git@marcbrinkmann.de>",
+]
+categories = ["algorithms", "asynchronous", "cryptography", "network-programming"]
+keywords = ["consensus", "asynchronous", "threshold"]
 license = "MIT/Apache-2.0"
-repository = "https://github.com/poanetwork/hbbft"
 readme = "README.md"
-keywords = ["consensus", "asynchronous", "threshold"]
-categories = ["algorithms", "asynchronous", "cryptography", "network-programming"]
+repository = "https://github.com/poanetwork/hbbft"
+description = "Honey Badger Byzantine fault tolerant consensus algorithm"
```

Comment:
Capitalization

Reply:
Good point. How about:
"The Honey Badger of Byzantine Fault Tolerant Consensus Protocols"?
That would also be closer to the paper's title.

Reply:
Yup :)